Source/Tanks/ModuleFuelTanks.cs

```diff
@@ -18,6 +18,8 @@ public class ModuleFuelTanks : PartModule, IModuleInfo, IPartCostModifier, IPart
 	{
 		bool compatible = true;
 
+        public readonly EventVoid onUpdateTankType = new EventVoid("onUpdateTankType");
```

Comment:
One other option here would be to use a part event.  That would less prone to NREs, and would mean that that the TF module doesn't need to know about the tank directly, but also harder to detect if it breaks.